website/app/styles/pages/components/table.scss

```diff
@@ -3,9 +3,9 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-// COMPONENTS > TABLE
+// COMPONENTS > TABLE > TABLE
 
-#show-content-components-table {
+#show-content-components-table-table {
```

Comment:
"It's a table table"? 😉

Reply:
Hahaha I know its not ideal. But with the way the class names are generated so im kinda 🤷🏽